usr.sbin/daemon/daemon.c

```diff
@@ -418,8 +418,8 @@ main(int argc, char *argv[])
 	}
 
 	/* else: pid > 0
-         * fork succeeded, this is parent branch, this can only when
-	 * supervision_enabled
+         * fork succeeded, this is the parent branch, this can only happen when
```

Comment:
Let's go ahead and squash this back into the commit that added the comment since that's still new to this PR

Reply:
ack

Reply:
done